### PR TITLE
Improve refresh script

### PR DESCRIPTION
**Pull request by szepeviktor**
# 🚀

### CODE REVIEW

File: bin/refresh-wp-install-tests.sh [hunk 1]
```diff
@@ -3,9 +3,9 @@
...
 
 SRC="https://raw.githubusercontent.com/wp-cli/scaffold-command/master/templates/install-wp-tests.sh"
-DIR="$( cd "$(dirname "$0")" || exit 1; pwd -P )"
+DIR="$(cd "$(dirname "$0")" || exit 1; pwd -P)"
 
 curl -s "$SRC" \
-    | sed $'2i\\\n# Notice: This file is pulled from the WP-CLI scaffold command,\\\n# and should not be modified directly.\n' \
-    > "$DIR/install-wp-tests.sh" \
-    && echo -e "\033[0;32mbin/install-wp-tests.sh has been updated to match ${SRC}\033[0;0m"
+    | sed -e '2 i # Notice: This file is pulled from the WP-CLI scaffold command,\n# and should not be modified directly.' \
```
Comment (szepeviktor) [inline bin/refresh-wp-install-tests.sh:9]: This way you have much less backslashes.

File: bin/refresh-wp-install-tests.sh [hunk 2]
```diff
@@ -3,9 +3,9 @@
...
+    > "${DIR}/install-wp-tests.sh" \
+    && printf '\033[0;32m%s %s\033[0;0m\n' "bin/install-wp-tests.sh has been updated to match" "${SRC}"
```
Comment (szepeviktor) [inline bin/refresh-wp-install-tests.sh:11]: Separate ANSI sequences from text.

**Comment by szepeviktor** [comment]
What do you think @stevegrunwell?

**Comment by stevegrunwell** [comment]
Hey @szepeviktor, thank you for this.

I actually have a branch locally in which I've fully rewritten the update script and am in the process of getting it integrated with GitHub Actions so that it can automatically issue a PR when new updates are made.

I'm going to hold on merging this one, but will integrate your changes wherever they might fit in my branch.

Thanks again!

**Pull request closed without merging**